swig/include/python/ogr_python.i

```diff
@@ -743,6 +743,11 @@ def ReleaseResultSet(self, sql_lyr):
 
         return self.GetGeometryRef()
 
+
+    def __repr__(self):
```

Comment:
maybe we need some tiny testing of this in autotest/ogr/ogr_feature.py ?

Reply:
I actually don't see any tests for `DumpReadableAsString`, so I'll plan to expose its "options" argument in SWIG and add some tests for that as well as `__repr__` ?

Reply:
> I actually don't see any tests for `DumpReadableAsString`

some of it might already be tested through test_ogrinfo.py with the --fields and --geom switches

> so I'll plan to expose its "options" argument in SWIG and add some tests for that as well as __repr__ ?

sounds good (I had only in mind minimal testing)